android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -36,8 +36,8 @@ class MusicService : HeadlessJsTaskService() {
     private val scope = MainScope()
     private var progressUpdateJob: Job? = null
 
-    var stoppingAppPausesPlayback = true
-        private set
+    private var stoppingAppAlsoStopsPlayback = false
+    private var stoppingAppRemovesNotification = false
```

Comment:
@mpivchev thoughts on making this default to `true`? Since it's only enabled if `stoppingAppAlsoStopsPlayback = true`, it seems like most people would want the notification to be removed as well. At least judging by all the feedback we've gotten.

Reply:
The people that didn't have this problem obviously never mentioned it, so the perception is a bit skewed. Still, we can make this true if everyone agrees that it's better.